media/css/firefox/switch.scss

```diff
@@ -7,8 +7,8 @@ $image-path: '/media/protocol/img';
 @import '../../protocol/css/includes/lib';
 @import '../../protocol/css/components/hero';
 
-@import '../hubs/sub-nav-protocol';
-@import '../hubs/sub-nav-download-button-remove-protocol';
+@import '../protocol/hubs/sub-nav';
```

Comment:
The /firefox/switch/ page doesn't seem to feature a sub nav any longer, so I think this line and below can be removed?